code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -220,9 +223,15 @@ Auto Patrol: []"},
 		return
 	if(iscarbon(A))
 		var/mob/living/carbon/C = A
-		if(!C.IsParalyzed() || arrest_type)
+		if(!C.IsParalyzed() || arrest_type || stuncount < 30)
```

Comment:
```suggestion
		if((!C.IsParalyzed() || arrest_type) && stuncount < 30)
```
This will add it as another condition, rather than overwriting the existing ones